Mathlib/Topology/Connected/Clopen.lean

```diff
@@ -121,6 +122,16 @@ theorem IsClopen.eq_univ [PreconnectedSpace α] {s : Set α} (h' : IsClopen s) (
     s = univ :=
   (isClopen_iff.mp h').resolve_left h.ne_empty
 
+open Set.Notation in
+lemma isClopen_preimage_val {X : Type*} [TopologicalSpace X] {u v : Set X}
+    (hu : IsOpen u) (huv : frontier u ∩ v = ∅) : IsClopen (v ↓∩ u) := by
```

Comment:
Is `frontier u ∩ v = ∅` the best way to state this? Do we not have a disjointness predicate?

Reply:
Thanks, good point, I'll make that change.